myselfのコピー.css

```diff
@@ -185,21 +190,18 @@ span{
 }
 
 .fgn{
```

Comment:
better to use more clear class names, it is easier for you to remember where this class name is used in the code :)